pom.xml

```diff
@@ -1171,6 +1172,19 @@ under the License.
 						</plugin>
 					</plugins>
 				</pluginManagement>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<excludedGroups>
```

Comment:
Can we have a separate commit that sets up surefire for this plugin (+link it in FLINK-33163), and then modify it for the GHA stuff in another one?